tests/api/test_cargo_movement.py

```diff
@@ -24,111 +18,101 @@ class TestCargoMovement(TestCase):
-                            **{
-                                "id": "2dfc3d43a21697c02ec3b2700b3b570a6ed1abb66d01c4fe6310ef362fcf6653",
-                                "layer": "country",
-                                "label": "Netherlands",
-                                "source": "model",
-                                "probability": 1,
-                            }
-                        )
-                    ],
-                    "probability": 1,
-                    "pos": [4.29914090037834, 51.87936163368058],
-                    "start_timestamp": "2019-10-18T21:38:34+0000",
-                    "end_timestamp": "2019-10-20T16:41:49+0000",
-                }
-            )
+            {
+                "event_type": "cargo_port_load_event",
+                "location": [
+                    {
+                        "id": "2dfc3d43a21697c02ec3b2700b3b570a6ed1abb66d01c4fe6310ef362fcf6653",
+                        "layer": "country",
+                        "label": "Netherlands",
+                        "source": "model",
+                        "probability": 1,
+                    }
+                ],
+                "vessel_id": "9cbf7c0fc6ccf1dfeacde02b87f3b6b1653030f560d4fc677bf1d7d0be8f8449",
+                "probability": 1,
+                "pos": [4.29914090037834, 51.87936163368058],
+                "start_timestamp": "2019-10-18T21:38:34+0000",
+                "end_timestamp": "2019-10-20T16:41:49+0000",
+            }
         ],
         "parent_ids": [
-            ParentID(
-                **{
-                    "id": "9d52ede1cff0421a8cd7283b0171afe8d23f519dca5f4e489734522f9cdf804c",
-                    "splinter_timestamp": "2019-10-20T16:41:49+0000",
-                }
-            )
-        ]
+            {
+                "id": "9d52ede1cff0421a8cd7283b0171afe8d23f519dca5f4e489734522f9cdf804c",
+                "splinter_timestamp": "2019-10-20T16:41:49+0000",
+            }
+        ],
     }
 
-    cm = CargoMovement(**dictionary)
+    cm = CargoMovement.parse_obj(dictionary)
 
     def test_serialize(self):
         with open("tests/api/examples/cargo_movements.json", "r") as f:
-            serialized = f.read()
-            deserialized = jsons.loads(serialized, List[CargoMovement])
+            serialized = json.load(f)[0]
+            deserialized = CargoMovement.parse_obj(serialized)
 
-            assert [self.cm] == deserialized
+            assert self.cm == deserialized
 
     def test_convert_to_flat_dict(self):
-        flat = convert_cargo_movement_to_flat_dict(serialize_to_dict(self.cm))
+        flat = convert_cargo_movement_to_flat_dict(self.cm.dict())
 
         expected = {
             "cargo_movement_id": "00886b05a0747522b67322f50123ee60e61e219fc9a9c6011be1a1dade65f63e",
             "events.cargo_port_load_event.0.end_timestamp": "2019-10-20T16:41:49+0000",
+            "events.cargo_port_load_event.0.end_timestamp": datetime.datetime(
```

Comment:
Do we need to create the `datetime` here?

Reply:
I've changed this back to a str.